docs/csharp/event-pattern.md

```diff
@@ -237,6 +195,17 @@ can also make the types used for the arguments internal as well.
 You'll start by creating the new EventArgs derived class for
 reporting the new directory and progress. 
 
+[!code-csharp[DirEventArgs](../../samples/csharp/events/Program.cs#SearchDirEventArgs "Define search directory event arguments")]
+
+Again, you can follow the recommendations to make an immutable
+reference type for the event arguments.
+
+Next, define the event. This time, you'll use a different syntax. In
+addition to using the field syntax, you can explicitly create the
+property, with add and remove handlers. In this sample, you won't
+need extra code in those handlers in this project, but this shows how
+you would create them.
+
 ```csharp
 internal class SearchDirectoryArgs : EventArgs
```

Comment:
SearchDirectoryArgs, or SearchDirectoryEventArgs?

Reply:
The code uses `SearchDirectoryArgs` throughout.